app/assets/javascripts/client/inventories/addInventoryUsersLink.js

```diff
@@ -0,0 +1,18 @@
+(function () {
+  'use strict';
+  angular.module('PDRClient')
+      .directive('addInventoryUsersLink', addInventoryUsersLink);
+
+  function addInventoryUsersLink() {
+    return {
+      restrict: 'E',
+      scope: {
+        inventoryId: '=inventoryId'
+      },
+      templateUrl: 'client/inventories/add_inventory_users_link.html',
+      controller: 'AddInventoryUsersLinkCtrl',
+      controllerAs: 'link',
```

Comment:
I'd prefer a slightly longer `controllerAs` name to this as opposed to "link" which is open to a lot of interpretation.  Something like `inventoryUsersLink`.